features/frameworks/utest/source/utest_harness.cpp

```diff
@@ -47,8 +47,15 @@ namespace
     size_t case_failed = 0;
     size_t case_failed_before = 0;
 
-    handlers_t defaults = default_handlers;
-    handlers_t handlers = defaults;
+    handlers_t& defaults() {
```

Comment:
Can handlers_t not be a class to and use the SingletonPtr paradigm for consistency?